include/translated-object.php

```diff
@@ -496,8 +496,9 @@ public function current_user_can_synchronize( $id ) {
 	 *
 	 * @since 3.2
 	 *
-	 * @param  mixed $id  A supposedly numeric ID.
-	 * @return int<0,max> A positive integer. `0` for non numeric values and negative integers.
+	 * @param mixed $id A supposedly numeric ID.
+	 * @return int A positive integer. `0` for non numeric values and negative integers.
+	 * @phpstan-return int<0,max>
```

Comment:
Like in https://github.com/polylang/polylang-pro/pull/1498#discussion_r997893752, we tend to add a blank line before the `@phpstan-*` tags apparently.
```suggestion
	 * @return int A positive integer. `0` for non numeric values and negative integers.
	 *
	 * @phpstan-return int<0,max>
```